server/src/main/java/org/bonitasoft/console/common/server/i18n/I18n.java

```diff
@@ -92,19 +147,21 @@ private String makeLocaleRegex(final LOCALE locale) {
     /**
      * @return available locale file for a specified application (i.e application_locale.po)
      */
-    private static File getLocaleFile(String locale, String application) {
-        return new File(I18N_DIR, application + "_" + locale + ".po");
+    private static boolean localeExists(String locale, String application) {
+        final String fileName = application + "_" + locale + ".po";
+        final File file = new File(I18N_CUSTOM_DIR, fileName);
+        return file.exists() || I18n.class.getResource("/i18n/" + fileName) != null;
     }
 
     @Override
     protected String getText(String string) {
         return string;
```

Comment:
I added tests for better coverage, and to make sure my changes did not break the behaviour